src/mobot_client/models/states.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
+
+from django.db import models
+
+
+class SessionState(models.IntegerChoices):
```

Comment:
I think using IntegerChoices here is inferior to using the string version. If we ever add another an intermediary step, this will get messy with anything that's already in the database, or the numbers will no longer be in increasing order.

Reply:
The reason we use it is because we wanted to use greater-than, less-than logic. Otherwise, I agree.

Reply:
I personally think that greater-than/less-than logic is a headache waiting to happen, and creating a list of possible states for that is encapsulated by the numbers range is the way to go (similar to `active_states`, `refundable_states,` etc). I realize this PR is not the place to fix it.